### PR TITLE
Dev to Master 

### PR DESCRIPTION
**Pull request by Antivortex**

### CODE REVIEW

File: Mirage-Android-WebView-Plugin/app/.gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+/build
```
Comment (Vukiz) [inline Mirage-Android-WebView-Plugin/app/.gitignore:1]: I don't think we need a second gitignore, right?

File: Mirage-Android-WebView-Plugin/app/src/main/java/com/mirage/miragewebviewandroidplugin/MirageWebViewActivity.java [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+        _webView.setWebViewClient(new CustomWebViewClient(_progressBar));
+        _webView.getSettings().setJavaScriptEnabled(true);
+        _webView.addJavascriptInterface(new CustomJavaScriptInterface(this), "AndroidInterface");
+
+        Intent intent = getIntent();
+        String url = intent.getStringExtra("url");
+        _webView.loadUrl(url);
+
+        _exitButton.setOnClickListener(v -> finish());
```
Comment (Vukiz) [inline Mirage-Android-WebView-Plugin/app/src/main/java/com/mirage/miragewebviewandroidplugin/MirageWebViewActivity.java]: extract finish into separate method.

File: Mirage-Android-WebView-Plugin/app/src/main/java/com/mirage/miragewebviewandroidplugin/MirageWebViewActivity.java [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+    public void onBackPressed() {
+        if(_webView.canGoBack()) {
+            _webView.goBack();
+        } else {
+            finish();
+        }
+    }
+
+    public void sendLoginDataToUnity(String loginData) {
+
```
Comment (Vukiz) [inline Mirage-Android-WebView-Plugin/app/src/main/java/com/mirage/miragewebviewandroidplugin/MirageWebViewActivity.java]: remove extra line

File: Mirage-Android-WebView-Plugin/app/src/main/java/com/mirage/miragewebviewandroidplugin/UnityBridge.java [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+import android.content.Context;
+import android.content.Intent;
+
+public class UnityBridge {
+    private static UnityBridge instance;
+    private Context context;
+
+    public UnityBridge(Context context) {
+        this.context = context;
```
Comment (Vukiz) [inline Mirage-Android-WebView-Plugin/app/src/main/java/com/mirage/miragewebviewandroidplugin/UnityBridge.java]: name it _context and you will no have to use this.

File: Mirage-Android-WebView-Plugin/app/src/test/java/com/mirage/miragewebviewandroidplugin/ExampleUnitTest.java [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+
+import static org.junit.Assert.*;
+
+/**
+ * Example local unit test, which will execute on the development machine (host).
+ *
+ * @see <a href="http://d.android.com/tools/testing">Testing documentation</a>
+ */
+public class ExampleUnitTest {
+    @Test
```
Comment (Vukiz) [inline Mirage-Android-WebView-Plugin/app/src/test/java/com/mirage/miragewebviewandroidplugin/ExampleUnitTest.java]: remove examples

File: Mirage-Widget/Packages/packages-lock.json [hunk 6]
```diff
@@ -0,0 +1,200 @@
...
+      "source": "builtin",
+      "dependencies": {}
+    },
+    "com.unity.modules.unitywebrequest": {
+      "version": "1.0.0",
+      "depth": 0,
+      "source": "builtin",
+      "dependencies": {}
+    },
+    "com.unity.modules.unitywebrequestassetbundle": {
```
Comment (Vukiz) [inline Mirage-Widget/Packages/packages-lock.json]: I think we should be able to remove a bunch of useless default packages from it, right?

File: Mirage-Widget/UserSettings/Search.settings [hunk 7]
```diff
@@ -0,0 +1 @@
+{}
```
Comment (Vukiz) [inline Mirage-Widget/UserSettings/Search.settings]: usersettings should be ignored

File: Mirage-Widget/ankr-sdk-renaming.py [hunk 8]
```diff
@@ -0,0 +1,68 @@
+import os
+
+def is_desired_extension(filename):
```
Comment (Vukiz) [inline Mirage-Widget/ankr-sdk-renaming.py]: this is not needed, right?

**Review by Vukiz — CHANGES_REQUESTED** [review]

**Review by Vukiz — DISMISSED** [review]

**Comment by Antivortex** [comment]
Now this PR contains all previously reviewed PRs with all of the changes requested in this PR

**Review by Vukiz — APPROVED** [review]

**Pull request merged**